manageiq-operator/api/v1alpha1/helpers/miq-components/httpd.go

```diff
@@ -659,9 +659,9 @@ func tlsSecretName(cr *miqv1alpha1.ManageIQ) string {
 	return secretName
 }
 
-func fetchIntrospectionUrl(providerUrl string) (string, error) {
+func fetchIntrospectionUrl(providerUrl string, sslVerify bool) (string, error) {
 	customTransport := http.DefaultTransport.(*http.Transport).Clone()
-	customTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	customTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: !sslVerify}
```

Comment:
This is fine, but can we also just not set it?  Something like

```suggestion
	if !sslVerify {
		customTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: !sslVerify}
	}
```

Reply:
Merging anyway.